core/src/test/resources/parameters_overloading.yaml

```diff
@@ -3,6 +3,21 @@ info:
   title: Projects API
   version: 1.0.0
 paths:
+  /projects/{key}:
```

Comment:
also there's a case like this
```
"/repos/{owner}/{repo}/git/commits/{sha}"
"/repos/{owner}/{repo}/git/commits/{sha}.{diffType}"
```

better to check this one also

Reply:
Did you mean "/" between {sha} and {diffType} ?

"/repos/{owner}/{repo}/git/commits/{sha}"
"/repos/{owner}/{repo}/git/commits/{sha}**/**{diffType}"

Reply:
Anyway, I've added a test case to be sure that such parameter {sha}.{diffType} will not clash with simple {sha}

Reply:
No. It is forgejo\gitea spec: https://codeberg.org/swagger.v1.json
you can find endpoints I mentioned there

Reply:
Thank you, I've added such a case to the test, it works well.